webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -45,6 +45,9 @@
         </v-card-text>
         <attach-image-component 
           :max-file-size="$root.maxFileSize"
+          :object-id="activityId"
+          :attached-files="attachments"
```

Comment:
Give the same name of variable in parent and child for code clearness 

```suggestion
          :attachments="attachments"
```